Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/feat'...
Listing './lib/python3.5/site-packages/feat/agencies'...
Compiling './lib/python3.5/site-packages/feat/agencies/bootstrap.py'...
***   File "./lib/python3.5/site-packages/feat/agencies/bootstrap.py", line 67
    opts.standalone_kwargs), None, sys.exc_info()[2]
                           ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/agencies/journaler.py'...
***   File "./lib/python3.5/site-packages/feat/agencies/journaler.py", line 160
    except error.FeatError, e:
                          ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/agencies/messaging'...
Listing './lib/python3.5/site-packages/feat/agencies/net'...
Compiling './lib/python3.5/site-packages/feat/agencies/net/options.py'...
***   File "./lib/python3.5/site-packages/feat/agencies/net/options.py", line 310
    except ImportError, e:
                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/agencies/net/ssh.py'...
***   File "./lib/python3.5/site-packages/feat/agencies/net/ssh.py", line 125
    % (agent_type, index, len(mediums))), None, sys.exc_info()[2]
                                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/agencies/replay.py'...
***   File "./lib/python3.5/site-packages/feat/agencies/replay.py", line 570
    except RuntimeError, e:
                       ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/agents'...
Listing './lib/python3.5/site-packages/feat/agents/alert'...
Listing './lib/python3.5/site-packages/feat/agents/base'...
Listing './lib/python3.5/site-packages/feat/agents/base/amqp'...
Compiling './lib/python3.5/site-packages/feat/agents/base/partners.py'...
***   File "./lib/python3.5/site-packages/feat/agents/base/partners.py", line 524
    raise ValueError('Unknown relation name %r: ' % (name, )), \
                                                             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/agents/base/resource.py'...
***   File "./lib/python3.5/site-packages/feat/agents/base/resource.py", line 252
    % (total, e, )), None, sys.exc_info()[2]
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/agents/common'...
Compiling './lib/python3.5/site-packages/feat/agents/common/start_agent.py'...
***   File "./lib/python3.5/site-packages/feat/agents/common/start_agent.py", line 74
    def _request_starting_host(self, state, (allocation_id, recp)):
                                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/agents/dns'...
Compiling './lib/python3.5/site-packages/feat/agents/dns/production.py'...
***   File "./lib/python3.5/site-packages/feat/agents/dns/production.py", line 114
    except error.CannotListenError, e:
                                  ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/agents/host'...
Listing './lib/python3.5/site-packages/feat/agents/integrity'...
Listing './lib/python3.5/site-packages/feat/agents/monitor'...
Listing './lib/python3.5/site-packages/feat/agents/nagios'...
Listing './lib/python3.5/site-packages/feat/agents/raage'...
Listing './lib/python3.5/site-packages/feat/agents/shard'...
Listing './lib/python3.5/site-packages/feat/common'...
Compiling './lib/python3.5/site-packages/feat/common/_noarch_run.py'...
***   File "./lib/python3.5/site-packages/feat/common/_noarch_run.py", line 69
    os.chmod(filename, 0644)
                          ^
SyntaxError: invalid token

Compiling './lib/python3.5/site-packages/feat/common/container.py'...
***   File "./lib/python3.5/site-packages/feat/common/container.py", line 670
    raise Empty(), None, sys.exc_info()[2]
                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/common/defer.py'...
***   File "./lib/python3.5/site-packages/feat/common/defer.py", line 99
    print _template % args
                  ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/feat/common/fiber.py'...
***   File "./lib/python3.5/site-packages/feat/common/fiber.py", line 127
    print _template % args
                  ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/feat/common/journal.py'...
***   File "./lib/python3.5/site-packages/feat/common/journal.py", line 180
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/common/manhole.py'...
***   File "./lib/python3.5/site-packages/feat/common/manhole.py", line 409
    def validate_result(self, (result_array, result_keywords, )):
                              ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/common/mro.py'...
***   File "./lib/python3.5/site-packages/feat/common/mro.py", line 66
    raise AttributeError(msg), None, sys.exc_info()[2]
                             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/common/resolver.py'...
***   File "./lib/python3.5/site-packages/feat/common/resolver.py", line 23
    def _cbRecords(self, (ans, auth, add), name, effort):
                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/common/run.py'...
***   File "./lib/python3.5/site-packages/feat/common/run.py", line 40
    print "%s is not running" % _describe(process_type, processName)
                            ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/common/serialization'...
Compiling './lib/python3.5/site-packages/feat/common/serialization/base.py'...
***   File "./lib/python3.5/site-packages/feat/common/serialization/base.py", line 494
    reflect.canonical_name(self), value)), \
                                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/configure'...
Listing './lib/python3.5/site-packages/feat/database'...
Compiling './lib/python3.5/site-packages/feat/database/client.py'...
***   File "./lib/python3.5/site-packages/feat/database/client.py", line 648
    def _log_update(self, ((_s1, doc), (_s2, tag), (_s3, seq)), update_log):
                          ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/database/couchdb'...
Compiling './lib/python3.5/site-packages/feat/database/couchdb/view.py'...
***   File "./lib/python3.5/site-packages/feat/database/couchdb/view.py", line 72
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/database/models.py'...
***   File "./lib/python3.5/site-packages/feat/database/models.py", line 235
    d.addCallback(lambda (r, _): r)
                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/database/view.py'...
***   File "./lib/python3.5/site-packages/feat/database/view.py", line 215
    exec code in {}, query_method.func_globals
            ^
SyntaxError: Missing parentheses in call to 'exec'

Listing './lib/python3.5/site-packages/feat/extern'...
Listing './lib/python3.5/site-packages/feat/extern/log'...
Compiling './lib/python3.5/site-packages/feat/extern/log/log.py'...
***   File "./lib/python3.5/site-packages/feat/extern/log/log.py", line 337
    except TypeError, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/extern/log/termcolor.py'...
***   File "./lib/python3.5/site-packages/feat/extern/log/termcolor.py", line 213
    print term.render('${BOLD}${RED}Error:${NORMAL}'), 'paper is ripped'
             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/extern/log/test_log.py'...
***   File "./lib/python3.5/site-packages/feat/extern/log/test_log.py", line 278
    except TypeError, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/extern/txamqp'...
Listing './lib/python3.5/site-packages/feat/extern/txamqp/contrib'...
Listing './lib/python3.5/site-packages/feat/extern/txamqp/contrib/thrift'...
Compiling './lib/python3.5/site-packages/feat/extern/txamqp/protocol.py'...
***   File "./lib/python3.5/site-packages/feat/extern/txamqp/protocol.py", line 109
    except QueueClosed, e:
                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/extern/txamqp/spec.py'...
***   File "./lib/python3.5/site-packages/feat/extern/txamqp/spec.py", line 219
    exec code in g, l
            ^
SyntaxError: Missing parentheses in call to 'exec'

Compiling './lib/python3.5/site-packages/feat/extern/txamqp/testlib.py'...
***   File "./lib/python3.5/site-packages/feat/extern/txamqp/testlib.py", line 111
    print "failed to connect to host: %s, port: %s; These tests are designed to run against a running instance" \
                                                                                                              ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/feat/gateway'...
Compiling './lib/python3.5/site-packages/feat/gateway/resources.py'...
***   File "./lib/python3.5/site-packages/feat/gateway/resources.py", line 197
    raise http.ForbiddenError, None, failure.tb
                             ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/interface'...
Listing './lib/python3.5/site-packages/feat/models'...
Compiling './lib/python3.5/site-packages/feat/models/action.py'...
***   File "./lib/python3.5/site-packages/feat/models/action.py", line 363
    except ValueError, e:
                     ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/models/applicationjson.py'...
***   File "./lib/python3.5/site-packages/feat/models/applicationjson.py", line 405
    except ValueError, e:
                     ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/models/value.py'...
***   File "./lib/python3.5/site-packages/feat/models/value.py", line 740
    except ValueError, e:
                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/process'...
Compiling './lib/python3.5/site-packages/feat/process/base.py'...
***   File "./lib/python3.5/site-packages/feat/process/base.py", line 225
    except socket.error, e:
                       ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/simulation'...
Listing './lib/python3.5/site-packages/feat/test'...
Compiling './lib/python3.5/site-packages/feat/test/common.py'...
***   File "./lib/python3.5/site-packages/feat/test/common.py", line 339
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/common_serialization.py'...
***   File "./lib/python3.5/site-packages/feat/test/common_serialization.py", line 375
    [int, long], [0L]),
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/test/data'...
Compiling './lib/python3.5/site-packages/feat/test/dummy_process.py'...
***   File "./lib/python3.5/site-packages/feat/test/dummy_process.py", line 44
    print "Written pid file to %s" % (path, )
                                 ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/test/integration'...
Compiling './lib/python3.5/site-packages/feat/test/integration/common.py'...
***   File "./lib/python3.5/site-packages/feat/test/integration/common.py", line 369
    print msg
            ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/feat/test/test_agencies_emu_agency.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_agencies_emu_agency.py", line 249
    def assertsOnAgency((guid, msg, )):
                        ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_agencies_journaler.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_agencies_journaler.py", line 276
    except FailTest, e:
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_agencies_journaler_postgres.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_agencies_journaler_postgres.py", line 41
    except psycopg2.Error, e:
                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_agencies_messaging_routing.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_agencies_messaging_routing.py", line 176
    def direct((key, shard), expiration_time=None):
               ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_common_enum.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_common_enum.py", line 47
    except TypeError, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_common_reflect.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_common_reflect.py", line 73
    except TypeError, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_common_serialization_adapters.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_common_serialization_adapters.py", line 99
    except TypeError, e:
                    ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_common_serialization_json.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_common_serialization_json.py", line 78
    yield [int, long], [0L], str, ["0"], False
                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_common_serialization_pytree.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_common_serialization_pytree.py", line 515
    yield long, [0L], long, [0L], False
                  ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_common_serialization_sexp.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_common_serialization_sexp.py", line 172
    yield long, [0L], long, [0L], False
                  ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/test/test_models_action.py'...
***   File "./lib/python3.5/site-packages/feat/test/test_models_action.py", line 176
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/utils'...
Listing './lib/python3.5/site-packages/feat/web'...
Compiling './lib/python3.5/site-packages/feat/web/httpserver.py'...
***   File "./lib/python3.5/site-packages/feat/web/httpserver.py", line 135
    except HTTPError, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat/web/markup'...
Compiling './lib/python3.5/site-packages/feat/web/security.py'...
***   File "./lib/python3.5/site-packages/feat/web/security.py", line 379
    % (p12_filename, e), cause=e), \
                                 ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/feat/web/webserver.py'...
***   File "./lib/python3.5/site-packages/feat/web/webserver.py", line 849
    except Exception, e:
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/feat-1.0.5.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/zope'...
Listing './lib/python3.5/site-packages/zope/interface'...
Listing './lib/python3.5/site-packages/zope/interface/common'...
Listing './lib/python3.5/site-packages/zope/interface/common/tests'...
Listing './lib/python3.5/site-packages/zope/interface/tests'...
Listing './lib/python3.5/site-packages/zope.interface-4.4.3.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
